demos/modipulate-gml/test/main.c

```diff
@@ -0,0 +1,200 @@
+/* Demo test app for modipulate-gml */
+
+#include <stdio.h>
+#include "modipulate_gml.h"
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
Got it. I copied this chunk from [the console demo](https://github.com/MrEricSir/Modipulate/blob/master/demos/modipulate/console/main.cpp#L6) but didn't see that macro there — should it be there as well?